docs/en/compute/table.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: global
+title: Alluxio Structured Data Service
+nickname: {{feature_name}}
+group: Data Applications
+priority: 7
+---
+{% assign feature_name="A GOOD FEATURE NAME" %}
+
+
+## Structured Data Services
+
+Alluxio 2.1.0 introduces a new service within Alluxio called {{feature_name}}.
+{{feature_name}} is a service for managing access to structured data, which serves a purpose similar to the
+[Apache Hive Metastore](https://cwiki.apache.org/confluence/display/Hive/Design#Design-Metastore)
+
+SQL engines, i.e. [Starburst Presto](https://starburstdata.com), leverage these metastore-like
+services to determine which, and how much data to read when executing queries.
+They store information about different database catalogs, tables, storage formats, data
+location, and more.
+{{feature_name}} is designed to make it simple and straightforward to retrieve and serve structured table
+metadata to these SQL engines.
+
+
+## Architecture
+
+{{feature_name}} is designed in a way very similar to the normal Alluxio filesystem.
+The service itself is not responsible for retaining all data, but is rather a caching service for
+metadata that originates in another location (i.e. MySQL, Hive).
+These are called **UDBs**.
```

Comment:
shall we call out that UDB stands for (Under Data Base)?

Reply:
updated